client/styles/components/_nav.scss

```diff
@@ -162,21 +162,26 @@
     position: absolute;
   }
 }
-.svg__logo g > path {
+
+.svg__logo {
+
   @include themify() {
-    fill: getThemifyVariable('logo-color');
+    // Set background color of the logo
+    background-color: getThemifyVariable('logo-color');
   }
-}
-.svg__logo g g:first-of-type path {
-  fill: none;
+
 }
 
-.svg__logo g g:first-of-type use {
+.svg__logo g path{
+  
   @include themify() {
+    // Set internal color of the logo;
```

Comment:
These comments will be helpful to anyone looking at this.  Because now that we inverted the SVG file the naming is kind of backwards isn't it!  `'logo-color'` is the background and `'logo-background-color'` is the text 😆 .